toptal/picasso
# refactor(Avatar): make avatar more reusable

### Description

- Refactor Avatar component so it is possible to be used inside AvatarGroup in [another PR](https://github.com/toptal/picasso/pull/2194). 
- Also fixed the size of clipped corner and Toptal logo so it complies with BASE 2.0

### How to test

- compare temploy with [design](https://app.abstract.com/projects/1b06c884-06af-482a-bf12-a82f521a19a1/branches/master/commits/c5c9dad3787a1f500fcbcc38f8ec74489b5a833b/files/6e64349b-3b5b-4af6-87a3-7f7d15b299ca/layers/1ACF9D6E-E04B-4D8E-AF99-0E655E250847?mode=build&selected=root-B0DC6F76-BCA2-4171-B246-A811CFAA25FC)
- change should have no impact for Avatar users, only internal changes

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![image](https://user-images.githubusercontent.com/6830426/139690164-e44dc22b-0895-491b-95d4-3205163dfc58.png) | ![image](https://user-images.githubusercontent.com/6830426/139689981-16cbb81d-6de3-4cd7-871f-67693b2dc008.png) |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
